src/test/kotlin/com/github/quillraven/fleks/ComponentTest.kt

```diff
@@ -142,26 +155,17 @@ internal class ComponentTest {
 
     @Test
     fun `do not create the same mapper twice`() {
-        val cmpService = ComponentService()
+        val cmpService = ComponentService(componentFactory)
         val expected = cmpService.mapper<ComponentTestComponent>()
 
         val actual = cmpService.mapper<ComponentTestComponent>()
 
         assertSame(expected, actual)
     }
 
-    @Test
-    fun `cannot create mapper for component without no-args constructor`() {
```

Comment:
This test case is obsolete with the constructor functions from the componentFactory.